Presentation.Web/Controllers/API/V2/External/ItSystems/ItSystemV2Controller.cs

```diff
@@ -393,6 +398,51 @@ public IHttpActionResult DeactivateSystemAsRightsHolder([NonEmptyGuid] Guid uuid
                 .Match(_ => StatusCode(HttpStatusCode.NoContent), FromOperationError);
         }
 
+        /// <summary>
+        /// Returns the permissions of the authenticated client in the context of a specific IT-System
+        /// </summary>
+        /// <param name="systemUuid">UUID of the system entity</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("it-systems/{systemUuid}/permissions")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ResourcePermissionsResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetItSystemPermissions([NonEmptyGuid] Guid systemUuid)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return _itSystemService
+                .GetPermissions(systemUuid)
+                .Select(_permissionResponseMapper.Map)
+                .Match(Ok, FromOperationError);
+        }
+
+
+        /// <summary>
+        /// Returns the permissions of the authenticated client for the IT-System resources collection in the context of an organization (IT-System permissions in a specific Organization)
+        /// </summary>
+        /// <param name="organizationUuid">UUID of the organization</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("it-systems/permissions/{organizationUuid}")]
```

Comment:
reading this route I would expect to retrieve a specific permission with a specific uuid

`systems/permissions/uuid`
Let's remove the uuid from the route param and move it to a required query parameter in stead. That makes the route make sense while not losing any functionality.. If the same design is applied in it-interfaces and it-system-usages, consider the same comment valid in those branches :-)